api/coupon.md

```diff
@@ -73,3 +73,4 @@ Error Code | HTTP Status Code | Description
 30         | 422              | Coupon quota exhausted.
 31         | 422              | User usage quota for this coupon is exhausted.
 32         | 422              | User usage quota for a coupon is same grouping is exhausted.
+40         | 422              | Platform is not enabled.
```

Comment:
please add some info about what a platform is, example would be okay.
Also keep in mind that this documentation should be readable for API end users, who will not immediately know what `enabled` means here, I would like for example:
`Platform(iOs, Android, ...) is not enabled. Some coupons only work on specific platform.`